packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -106,6 +125,11 @@ class Send extends Component {
                             step={0.5}
                             type='range' // In Gwei
                           />
+
+                          <div className='hidden'>
```

Comment:
You actually don't need this anymore, the estimated gas will always be in `values.gas` with the calculate extension.

Reply:
great, I didn't guess that